### ISSUE TITLE
Add DoE header to each file

### ISSUE THREAD

**Issue by sterlingbaldwin** [ACME-OUI/acme-web-fe#117]
We really should add the DoE official header to each file in the repo.

**Comment by chaosphere2112** [comment]
Should we? We don't have that in UV-CDAT.

**Comment by sterlingbaldwin** [comment]
It came up in the meeting, and it seemed to be the unanimous consensus that we should. Mostly Dean said we should, which is the important thing.

**Comment by chaosphere2112** [comment]
OK, sounds fair enough :smile:

**Comment by mattben** [comment]
Yes we need to first receive a ACME/DOE header from the council. Then add it to all files...
